src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -39,14 +39,14 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
     /**
      * Use the new allele frequency / QUAL score model
      */
-    @Argument(fullName = "useNewAFCalculator", shortName = "newQual", doc = "If provided, we will use the new AF model instead of the so-called exact model", optional = true)
+    @Argument(fullName = "use-new-qual-calculator", shortName = "newQual", doc = "If provided, we will use the new AF model instead of the so-called exact model", optional = true)
```

Comment:
This one (and a few other common/"classic" HC args) might possibly deserve to keep their short names, but I defer to @vdauwera on whether the short names should be refactored to a different style.